cmd/scouting-backend/main.go

```diff
@@ -44,14 +44,13 @@ func main() {
 		os.Exit(1)
```

Comment:
Would this be better as `"unable to connect to postgres server: %v\n"`? I keep reading this as if the error was the options...

Reply:
Yeah. That's a bit confusing. Will fix 👍